### DISCUSSION TITLE
Frigate Media clips and recordings in Home Assistant HACS.

### DISCUSSION THREAD

**Discussion by COLG81** [blakeblackshear/frigate#14893] [Ask A Question]
A few days ago, my media clips and recordings will no longer play through HACS addon in media browser Home Assistant. They still show up with thumbnails but will not play. Any ideas will be greatly appreciated.

**Comment by NickM-27** [comment]
Do they work in the frigate UI?

**Reply by COLG81** [reply]
Yes, they do.

**Reply by NickM-27** [reply]
this would be something to post on the integration repo, I believe there is already an open issues for that though
